firefox-ios/Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -74,24 +78,34 @@ class Setting: NSObject {
         cell.detailTextLabel?.assign(attributed: status, theme: theme)
         cell.detailTextLabel?.attributedText = status
         cell.detailTextLabel?.numberOfLines = 0
-        cell.textLabel?.assign(attributed: title, theme: theme)
-        cell.textLabel?.textAlignment = textAlignment
-        cell.textLabel?.numberOfLines = 0
-        cell.textLabel?.lineBreakMode = .byTruncatingTail
-        cell.accessoryType = accessoryType
-        cell.accessoryView = accessoryView
-        cell.selectionStyle = enabled ? .default : .none
-        cell.accessibilityIdentifier = accessibilityIdentifier
-        cell.imageView?.image = _image
-        if let title = title?.string {
+        if let centeredTitle {
+            centeredLabel.assign(attributed: centeredTitle, theme: theme)
+            centeredLabel.textAlignment = textAlignment
```

Comment:
Do we still need to set the text alignment here? I think for the centered style the cell should handle this.